spec/unit/app/models/factcheck_outcome_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe FactcheckOutcome, type: :model do
+  let(:creator) { create(:user) }
+  let(:outcome) { described_class.new("creator" => creator) }
+
+  it "should get and set the reviewer" do
```

Comment:
under the hood this is actually storing this data in the reviewer_identifier field, but I didn't want to test that directly as that feels brittle and too tightly coupled to the implementation. I just care about the outcome.